x-pack/plugins/maps/public/components/toolbar_overlay/set_view_control/index.js

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { connect } from 'react-redux';
+import { SetViewControl } from './set_view_control';
+import { setGotoWithCenter } from '../../../actions/store_actions';
+import { getMapZoom, getMapCenter } from '../../../selectors/map_selectors';
+import {
+  getIsSetViewOpen,
+  closeSetView,
+  openSetView,
+} from '../../../store/ui';
+
+function mapStateToProps(state = {}) {
+  return {
+    isSetViewOpen: getIsSetViewOpen(state),
+    zoom: getMapZoom(state),
+    center: getMapCenter(state),
+  };
+}
+
+function mapDispatchToProps(dispatch) {
+  return {
+    onSubmit: ({ lat, lon, zoom }) => {
+      dispatch(closeSetView());
+      dispatch(setGotoWithCenter({ lat, lon, zoom }));
+    },
+    closeSetView: () => {
+      dispatch(closeSetView());
+    },
+    openSetView: () => {
+      dispatch(openSetView());
+    }
+  };
+}
+
+const connectedSetViewControl = connect(mapStateToProps, mapDispatchToProps, null, { withRef: true })(SetViewControl);
```

Comment:
Can remove 3rd and 4th args here since we aren't using refs in the component. Prob a copy/paste error. Update to: `const connectedSetViewControl = connect(mapStateToProps, mapDispatchToProps)(SetViewControl);`